proxy/src/http/control.rs

```diff
@@ -146,13 +160,25 @@ mod handler {
     }
 
     /// Reset the coco state by creating a new temporary directory for the librad paths.
-    pub async fn nuke_coco(librad_paths: Arc<RwLock<Paths>>) -> Result<impl Reply, Rejection> {
-        let dir = tempfile::tempdir().expect("tmp dir creation failed");
-        let new = Paths::from_root(dir.path()).expect("unable to get paths");
+    pub async fn nuke_coco(peer: Arc<Mutex<coco::Peer>>) -> Result<impl Reply, Rejection> {
+        // TmpDir deletes the temporary directory once it DROPS.
+        // This means our new directory goes missing, and future calls will fail.
+        // The Peer creates the directory again.
+        //
+        // N.B. this may gather lot's of tmp files on your system. We're sorry.
```

Comment:
LOL. So the temp files will never be cleaned up after the tests finish?

Reply:
Nope... Only when you reset your computer :innocent:

Reply:
I see. And where do these files get created when we run it on CI?

Reply:
Probably under `/tmp` or `/run/user/1000/.tmp*`. These get cleared out by the OS upon restart, afaik :thinking:

Reply:
I guess on CI that would be in the docker image which gets thrown away after the tests run. So should be okay.